store/store.go

```diff
@@ -595,9 +595,12 @@ type GroupStore interface {
 	TeamMembersToRemove() StoreChannel
 	ChannelMembersToRemove() StoreChannel
 
-	GetGroupsByChannel(channelId string, page, perPage int) StoreChannel
+	GetGroupsByChannel(channelId string, opts model.GroupSearchOpts) StoreChannel
```

Comment:
Not for now, but if you going to add new store endpoints, would be great if you add them using the new "sync by default" approach :)